pkg/apis/elasticsearch/v1alpha1/types.go

```diff
@@ -56,6 +57,28 @@ type ElasticsearchNode struct {
 	Storage      ElasticsearchNodeStorageSource `json:"storage"`
 }
 
+type ElasticsearchStorageSpec struct {
+	LocalVolume *ElasticsearchLocalVolumeSpec `json:"localVolume,omitempty"`
+	StorageClass *ElasticsearchStorageClassSpec `json:"storageClass,omitempty"`
+
+	// PersistentVolumeClaim will NOT try to regenerate PVC, it will be used
+	// as-is. You may want to use it instead of VolumeClaimTemplate in case
+	// you already have bounded PersistentVolumeClaims you want to use, and the names
+	// of these PersistentVolumeClaims doesn't follow the naming convention.
+	PersistentVolumeClaim *v1.PersistentVolumeClaimVolumeSource `json:"persistentVolumeClaim,omitempty"`
```

Comment:
Thinking about this further, how is this matched up for a multinode ES cluster?  Should this be a list of possible PVC? Should this be a prefix with the understanding that there N PVCs with a known naming convention?

Reply:
No, the storage definition is per node

Reply:
If we use the def per node, then we can not pull it directly into cluster-logging-operator CR.  That CR does not provide the ability to define storage attributes for individual members in an ES cluster.